bin/collator/src/local/chain_spec.rs

```diff
@@ -117,8 +117,8 @@ fn testnet_genesis(
         },
         block_reward: BlockRewardConfig {
             // Make sure sum is 100
-            reward_config: pallet_block_reward::RewardDistributionConfig {
-                base_treasury_percent: Perbill::from_percent(25),
+            reward_config: block_rewards_hybrid::RewardDistributionConfig {
+                treasury_percent: Perbill::from_percent(25),
```

Comment:
This is a good opportunity to get rid of the `pallet-block-reward` dependency in the collator code.

I'd suggest to just copy/paste the required fields from the `RewardDistributionConfig` into the genesis struct. From what I've seen, this is how most pallets in substrate do it.

Reply:
I don't get your suggestion. `RewardDistributionConfig` and `BlockRewardConfig` still need to be imported and have a dep towards `block_rewards_hybrid` pallet.

Reply:
`BlockRewardConfig` is imported from the `local-runtime`, not the pallet itself.

If you were to define genesis config as:
```
  pub struct GenesisConfig {
        pub treasury: Permill,
        pub collators: Permill,
       ...
    }
```
then you won't need a dependency towards the pallet in the client code since you won't need `RewardDistributionConfig`.

You could also try and use `pub use` in `local-runtime` to re-export the current implementation of `RewardDistributionConfig`, eliminating the need for the direct pallet dependency.

Reply:
I chose the second option as it's closer to how other types are imported and still ensuring use the correct type